scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -76,8 +76,8 @@ import org.ossreviewtoolkit.utils.showStackTrace
 import org.ossreviewtoolkit.utils.storage.FileArchiver
 
 /**
- * Implementation of [Scanner] for scanners that operate locally. Packages passed to [scanPackages] are processed in
- * serial order. Scan results can be stored in a [ScanResultsStorage].
+ * Abstraction for a [Scanner] that operates locally. Packages passed to [scanPackages] are processed in serial order.
```

Comment:
Should we remove the second sentence? It doesn't seem necessary to define here in what order packages are processed.

Reply:
Done.